packages/block-editor/src/components/block-alignment-control/ui.js

```diff
@@ -49,19 +49,21 @@ const POPOVER_PROPS = {
 	isAlternate: true,
 };
 
-export function BlockAlignmentToolbar( {
+function BlockAlignmentUI( {
 	value,
 	onChange,
 	controls = DEFAULT_CONTROLS,
+	isToolbar,
```

Comment:
Do we plan to remove this once we transition?

Reply:
I'm not sure we can, we have a valid use-case for it at the moment: The inspector control of the latest posts block. Also, it might have some impact on third party blocks.